content/en/docs/ibm/deploy/authentication.md

```diff
@@ -135,6 +135,10 @@ of the column `Hostname` in step 4:
           privateKey: /etc/istio/ingressgateway-certs/tls.key
           serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
     ```
+    
+    ```shell
+    kubectl apply -f kubeflow-gateway.yaml -n kubeflow
+    ```
```

Comment:
Thanks!

Do you mind adding some explanation saying "Run this command to apply changes to the gateway `kubeflow-gateway`:"

Reply:
sure will do

Reply:
@shawnzhu updated .. see if looks good now?